.github/workflows/unit-test-on-pull-request.yml

```
@@ -67,6 +67,28 @@ jobs:
       - name: Tests
         run: make test TARGET_ARCH=${{ matrix.target_arch }}
 
+  check-ebpf-blobs:
+    name: Check for differences in the eBPF binary blobs
+    runs-on: ubuntu-24.04
+    container: otel/opentelemetry-ebpf-profiler-dev:latest
+    steps:
+      - name: Clone code
+        uses: actions/checkout@v4
+      - name: Hash eBPF blobs
+        run: |
+          sha256sum support/ebpf/tracer.ebpf.release.a* > ebpf-blobs.hash
```

Just to keep it future proof (for example, for when we get RISCV or other hardware support)
```suggestion
          sha256sum support/ebpf/tracer.ebpf.release.* > ebpf-blobs.hash
```

---

resolving with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/305/commits/0d55b73dd09842b497e1c0dd110ba984b914e402